product_to_bizerba_scale/__manifest__.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2016-Today GRAP (http://www.grap.coop)
+# © 2017 Coop IT Easy (http://www.coopiteasy.be)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# @author: Houssine BAKKALI (https://github.com/houssine78)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
Could you add coop it easy copy right on all the py and limite the one from la louve to 2018.
Also make it comply with the CIE coopyright header.
thanks !

Reply:
done :)